Kitodo/src/main/java/org/kitodo/production/forms/SearchResultForm.java

```diff
@@ -156,6 +158,22 @@ private void refreshFilteredList() {
         filteredList.addAll(resultList);
     }
 
+    /**
+     * delete process.
+     * @param processDTO to delete
+     */
+    public void delete(ProcessDTO processDTO){
+       try {
+           ProcessForm processForm = new ProcessForm();
+           processForm.setProcess(ServiceManager.getProcessService().getById(processDTO.getId()));
+           processForm.delete();
+       }
+        catch (DAOException e) {
+            Helper.setErrorMessage("errorOnSearch", searchQuery);
+        }
+       filteredList.remove(processDTO);
```

Comment:
Is it a good idea to remove this process from the `filteredList` if deleting is causing a `DAOException`?